proto/cosmos/gov/module/v1/module.proto

```diff
@@ -10,10 +10,18 @@ message Module {
     go_import: "cosmossdk.io/x/gov"
   };
 
+  // max_title_len defines the maximum proposal title length.
+  // Defaults to 100 if not explicitly set.
```

Comment:
If you wish this to be backported to v0.50, please change the default to 255, otherwise we cannot backport it and that will get in 0.51.

Reply:
Good point, changes applied!